src/components/activities/ActivityCard.jsx

```diff
@@ -90,7 +136,15 @@ class ActivityCard extends Component {
             <span className='activity__date'>{date}</span>
           </div>
           <div className='activity__content'>
-            <h1 className='activity__description'>{description}</h1>
+            <p className='activity__description'>{description}
+              {
+                showMoreLess
```

Comment:
Consider setting this within before the return in the render function i.e :

```
let showMoreLessHtml = "";
if (showMoreLess){
showMoreLessHtml = // the html you want to render
}
```

and then place the `showMoreLessHtml` variable here. This is noted as better practice and looks cleaner in the long run when you have more similar inferences in your codebase.

Reply:
@kimani-ndegwa Working on it.